models/demos/falcon7b_common/tests/unit_tests/test_falcon_matmuls_and_bmms_with_mixed_precision.py

```diff
@@ -407,14 +407,11 @@ def test_falcon7b_attnention_sliced(
             compute_kernel_config=compute_kernel_config,
         )
 
-        mm_slice = ttnn.experimental.operations.primary.bcast(
-            mm_slice,
-            reference_scalar,
-            ttnn.experimental.tensor.BcastOpMath.MUL,
-            ttnn.experimental.tensor.BcastOpDim.HW,
-            output_mem_config=height_sharded_memory_config,
-            in_place=True,
-        )
+        # This should be done in-place.  This will be done by using an optional output tensor passed to multiply
+        # but this feature is being done on a separate effort.
+        mm_slice_temp = mm_slice
+        mm_slice = ttnn.multiply(mm_slice_temp, reference_scalar, memory_config=height_sharded_memory_config)
+        mm_slice_temp.deallocate()
```

Comment:
Why does this multiply use a temp copy for the input, while the others don't?

Reply:
Fixing this now...